tests/allpackages/TestAllIndividualPackages.cs

```diff
@@ -170,6 +177,17 @@ async Task TestAllPackages (string template, bool isGps)
 			return;
 		}
 
+		XmlDocument xd = new ();
```

Comment:
What is the goal here?

The lines later on like:

```csharp
ReplaceInFile (proj_file, $";{net_version}-ios", "");
```

should be removing anything that isn't Android.

If that is no longer working then we should probably fix it instead.